app-modules/ai/src/Observers/PromptObserver.php

```diff
@@ -36,12 +36,36 @@
 
 namespace AdvisingApp\Ai\Observers;
 
+use Laravel\Pennant\Feature;
 use AdvisingApp\Ai\Models\Prompt;
+use Illuminate\Support\Facades\Cache;
 
 class PromptObserver
 {
     public function creating(Prompt $prompt): void
     {
         $prompt->user()->associate(auth()->user());
     }
+
+    public function saved(Prompt $prompt): void
+    {
+        if (Feature::active('ai_utilization') && Cache::has('promt_by_category_chart')) {
+            Cache::forget('promt_by_category_chart');
+        }
+
+        if (Feature::active('ai_utilization') && Cache::has('prompts_created_line_chart')) {
+            Cache::forget('prompts_created_line_chart');
+        }
```

Comment:
No need for `Cache::has` and these two if statements could probably be combined.

Reply:
done